crates/router/src/connector/payme/transformers.rs

```diff
@@ -198,14 +198,15 @@ impl<F, T>
     fn try_from(
         item: types::ResponseRouterData<F, PaymePaySaleResponse, T, types::PaymentsResponseData>,
     ) -> Result<Self, Self::Error> {
-        let response = if item.response.sale_status == SaleStatus::Failed {
+        let status = enums::AttemptStatus::from(item.response.sale_status.clone());
```

Comment:
You pulled this status out, but you are not using it again, like in other files you are using it to populate `attempt_status` is this a miss?

Reply:
I am using this status to pass to `is_payment_failure` function as well.
`let response = if is_payment_failure(status) {`